doc/source/schemas/sequenceAnnotationmethods.rst

```diff
@@ -0,0 +1,457 @@
+SequenceAnnotationMethods
```

Comment:
@reece because Feature (discrete annotations) are not going to
be the only type of sequence annotation.  There is also be
contiguous (wiggle).  There is a proposed data structure,
however, it's not implemented and hence not included in this
release.


Reece Hart <notifications@github.com> writes:
> In doc/source/schemas/sequenceAnnotationmethods.rst:
>     The central object of the GA4GH Sequence Annotation API is a Feature.
> 
> I think this underscores the point I'm trying to make. If the core concept is a
> Feature, then why isn't this the Sequence Feature API?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub*
>